documentation/docs/analytics.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 8
```

Comment:
Also increment the position of the documentation pages that come after this one :
- Upgrading from v3
- Packages index
- v3 docs index